theano/gpuarray/subtensor.py

```diff
@@ -389,7 +389,7 @@ def add_to_zview(self, nodename, x, fail):
           void *args[2];
           args[0] = &zview->ga;
           args[1] = &%(x)s->ga;
-          if (GpuElemwise_call(iadd, args, GE_BROADCAST) != GA_NO_ERROR) {
+          if (GpuElemwise_call(iadd, args, GE_BROADCAST | GE_PADSHAPE) != GA_NO_ERROR) {
```

Comment:
Note, @abergeron added the python interface for this. So we can now update the perform() with the same fix in the 2 ops.

Also, if we want to merge this earlier, we could do the padshape in the graph or in the perform/c_code. I don't suggest doing that, just in case you find the idea useful or if we start to not be able to wait for the 0.6.10 release.